src/cc-index.ts

```diff
@@ -29,10 +28,7 @@ export class CCIndex {
     public defaultCCIndex : string;
 
     constructor(crawlVersion? : string) {
-        if (!crawlVersion) {
```

Comment:
Well, this only sets the default crawl index. If you don't supply it in the constructor you have to supply it with every function call... I don't see a problem here